Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM ruby:2.3.1
+RUN apt-get update -qq && apt-get upgrade -y
+
+RUN apt-get install -y build-essential nodejs && apt-get clean
+
+ENV GOVUK_APP_NAME publisher
+ENV MONGODB_URI mongodb://mongo/govuk_content_development
+ENV TEST_MONGODB_URI mongodb://localhost/govuk_content_publisher_test
+ENV PORT 3006
```

Comment:
Publisher normally runs on port 3000: https://github.com/alphagov/govuk-puppet/blob/31cd78613e34669305f781eed082d2f75e00c0cc/development-vm/Procfile#L1

Hopefully that doesn't cause problems given its common usage

Reply:
Thanks for the link 👍 .  I'll use 3000 to reduce potential confusion.